packages/components/progress/src/circle-progress.tsx

```diff
@@ -78,11 +90,15 @@ export type CircleProgressProps = Omit<HTMLUIProps<"div">, "color"> &
  * @see Docs https://yamada-ui.com/components/feedback/circle-progress
  */
 export const CircleProgress = forwardRef<CircleProgressProps, "div">(
-  (
-    {
+  ({ size: boxSize = "6rem", ...props }, ref) => {
```

Comment:
Please change `({ size: boxSize = "6rem", ...props }, ref) => {` to `(props, ref) => {`.